bpm/bonita-core/bonita-process-engine/src/main/java/org/bonitasoft/engine/api/impl/transaction/actor/ImportActorMapping.java

```diff
@@ -10,22 +10,24 @@
  * You should have received a copy of the GNU Lesser General Public License along with this
  * program; if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth
  * Floor, Boston, MA 02110-1301, USA.
- **/
+ ******************************************************************************/
+
 package org.bonitasoft.engine.api.impl.transaction.actor;
 
 import java.io.IOException;
-import java.io.StringReader;
 import java.util.List;
 import java.util.Set;
 
+import javax.xml.bind.JAXBException;
```

Comment:
What I meant is that you can raise an exception associated to `ActorMappingConverter`, let's say `ActorMappingConverterException' (declared in same module than ActorMappingConverter) so you don't have JAXBException and SaxException bubbling over the whole code but only ActorMappingConverterException which wraps Serialization/Deserialisation Exception.

But, again, I don't have the whole picture and I don't know what it's imply